src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaQueryBuilderTest.java

```diff
@@ -56,7 +56,7 @@ void createQuery_should_return_null_when_orQueries_is_empty_but_has_order_by() {
         SearchTaskRequest searchTaskRequest = new SearchTaskRequest(
             emptyList(),
             singletonList(
-                new SortingParameter(SortField.DUE_DATE, SortOrder.DESCENDANT)
+                new SortingParameter(SortField.DUE_DATE_CAMEL_CASE, SortOrder.DESCENDANT)
             ));
```

Comment:
Can we also add a test to cover DUE_DATE_SNAKE_CASE?

Reply:
done